simulator-core/build.gradle

```diff
@@ -31,8 +31,14 @@ lombok {
     sha256 = "39f3922deb679b1852af519eb227157ef2dd0a21eec3542c8ce1b45f2df39742"
 }
 
+configurations {
+    internalDependencies
+}
+
+configurations.compile.extendsFrom(configurations.internalDependencies)
+
 dependencies {
-    compile project(':payload:ocpp20')
+    internalDependencies project(':payload:ocpp20')
```

Comment:
is it possible to release payload pojos as a separate artifact?

Reply:
yes, but then every time you change them you will have to re-release them.

I would prefer to wait until we have versioning and continous release (https://myevbox.atlassian.net/browse/SPE-833?oldIssueView=true)

Reply:
As far as I know all pojos are there, so we may release it only once. It can be also used on a server-side